### ISSUE TITLE
flycheck:  javascript-flow is not a valid syntax checker 

### ISSUE THREAD

**Issue by RenWenshan** [an-sh/flow-minor-mode#20]
Got:
> flycheck-add-mode: javascript-flow is not a valid syntax checker
> File mode specification error: (error javascript-flow is not a valid syntax checker)

My configuration:
```emacs-lisp 
 (require 'flow-minor-mode)
 (add-hook 'js-mode-hook 'flow-minor-enable-automatically)
  (add-hook 'js2-mode-hook 'flow-minor-enable-automatically)

  (with-eval-after-load 'flycheck
    (flycheck-add-mode 'javascript-flow 'flow-minor-mode)
    (flycheck-add-mode 'javascript-eslint 'flow-minor-mode)
    (flycheck-add-next-checker 'javascript-flow 'javascript-eslint))
```

Environment: Spacemacs 0.300.0@26.1 on MacOS

**Comment by an-sh** [comment]
this is a problem with javascript-flow, which should be installed separately and is not a part of this minor mode

**Comment by RenWenshan** [comment]
@an-sh thanks for your reply. May I know where I can find `javascript-flow` (I assume it's an emacs plugin)?

**Comment by olessavluk** [comment]
@RenWenshan you need to install and then require it before using
```
(require 'flycheck-flow)
...
```

**Comment by sgrove** [comment]
@olessavluk The question is, where can we find the `javascript-flow` package? Is there a melpa package, or a git repo somewhere?

**Comment by olessavluk** [comment]
@sgrove yes, it is available on MELPA, but under different name [`flycheck-flow`](https://github.com/olessavluk/ohai-emacs/blob/master/modules/ohai-flow.el)

Here is [configuration that I use](https://github.com/olessavluk/ohai-emacs/blob/master/modules/ohai-flow.el) for Flow, hope it helps.
